### DISCUSSION TITLE
I would like to be a contributor

### DISCUSSION THREAD

**Discussion by HicaroD** [yassinebridi/serpl#40] [General]
I went to the documentation and there is no guide for contributors. I was about to build a similar tool and I just find yours. I love the idea of having a search and replace tool in my terminal, especially when the UI is really clean and easy to interact with.

I don't how your development process works, but I would like to understand it and help you to keep this project going, if you want me to.

**Comment by HicaroD** [comment]
I didn't want to create an issue just for this, I think it would be akward. That's why I wrote a a general discussion, not only for me, but also for any other people that wants to contribute to your project.

**Comment by yassinebridi** [comment]
Hey @HicaroD, i'm really thankful you considered this, since i don't have enough time to maintain this on my own.

There is no contributing guide, nor there is testing at the moment.

What we can do for now is:
- We can discuss something you want to work on
- Work on it
- Manual testing on the environments we have
- Make sure somethings are good (eg: documentation)
- Ship

**Reply by HicaroD** [reply]
https://github.com/yassinebridi/serpl/issues/22

I think this is quite important. Are you still planning to implement this feature? I could work on it!

**Reply by yassinebridi** [reply]
Yes please, would love if you picked that up.
